PurchasesTests/Purchasing/StoreKitRequestFetcherTests.swift

```diff
@@ -11,10 +11,9 @@ import OHHTTPStubs
 import Nimble
 import StoreKit
 
-import Purchases
+@testable import PurchasesCoreSwift
 
-
-class StoreKitRequestFetcher: XCTestCase {
+class StoreKitRequestFetcherTests: XCTestCase {
```

Comment:
this drove me nuts for a bit - the test class was named the same as the Swift class 🤦 It wasn't an issue before because it used to translate to `RCStoreKitRequestFetcher` before